travharv/web_discovery.py

```diff
@@ -90,31 +90,24 @@ def get_description_into_graph(
     session.mount("http://", adapter)
     session.mount("https://", adapter)
 
-    formats = ["text/turtle", "application/ld+json"]
-    ACCEPTABLE_MIMETYPES = {
-        "application/ld+json",
-        "text/turtle",
-        "application/json",
-    }
     triples_found = False
 
-    for format in formats:
```

Comment:
why did this go?

I was expecting formats (plural) to be added to the signature as an argument, not having this working strategy be removed like this

do not understand the reasoning